src/app/atoms/FormField.tsx

```diff
@@ -144,14 +144,14 @@ export const FormField = forwardRef<FormFieldElement, FormFieldProps>(
 
     const handleSecretBannerClick = () => void spareRef.current?.focus();
     const handleCleanClick = useCallback(() => void onClean?.(), [onClean]);
-
+    
     return (
       <div
         className={classNames('w-full flex flex-col', containerClassName)}
         style={containerStyle}
         {...setTestID(testIDs?.inputSection)}
       >
-        {label && (
+        {label !== "Password" && (
```

Comment:
it's an atom component FormField that is used across the app, better to handle this scenario outside the component

F.e pass empty label when it's "Password"